src/Microsoft.DotNet.XHarness.Common/Logging/ILog.cs

```diff
@@ -2,28 +2,15 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
The more I think about it the more I think it should go directly above `namespace` 😁 But here is fine as well, just above the license seemed weird